content/open-source/sigstore/cosign/how-to-verify-file-signatures-with-cosign.md

```diff
@@ -15,6 +18,8 @@ weight: 006
 toc: true
 ---
 
+/open-source/sigstore/rekor/how-to-verify-file-signatures-with-rekor-or-curl/
```

Comment:
This seems like an artifact? Should be removed?

Reply:
Oh gosh too many copy / pastes